challenges/08-coding-interview-prep/rosetta-code.json

```diff
@@ -6872,6 +6872,506 @@
+      ],
+      "tests": [
+        {
+          "text": "'<code>leftFactorial</code> should be a function.'",
+          "testString":
+            "assert(typeof leftFactorial=='function','<code>leftFactorial</code> should be a function.');"
+        },
+        {
+          "text": "'<code>leftFactorial(0)</code> should return a number.'",
+          "testString":
+            "assert(typeof leftFactorial(0)=='number','<code>leftFactorial(0)</code> should return a number.');"
+        },
+        {
+          "text":
+            "'<code>leftFactorial(0)</code> should return <code>0</code>.'",
+          "testString":
+            "assert.equal(leftFactorial(0),0,'<code>leftFactorial(0)</code> should return <code>0</code>.');"
+        },
+        {
+          "text":
+            "'<code>leftFactorial(1)</code> should return <code>1</code>.'",
+          "testString":
+            "assert.equal(leftFactorial(1),1,'<code>leftFactorial(1)</code> should return <code>1</code>.');"
+        },
+        {
+          "text":
+            "'<code>leftFactorial(2)</code> should return <code>2</code>.'",
+          "testString":
+            "assert.equal(leftFactorial(2),2,'<code>leftFactorial(2)</code> should return <code>2</code>.');"
+        },
+        {
+          "text":
+            "'<code>leftFactorial(3)</code> should return <code>4</code>.'",
+          "testString":
+            "assert.equal(leftFactorial(3),4,'<code>leftFactorial(3)</code> should return <code>4</code>.');"
+        },
+        {
+          "text":
+            "'<code>leftFactorial(10)</code> should return <code>409114</code>.'",
+          "testString":
+            "assert.equal(leftFactorial(10),409114,'<code>leftFactorial(10)</code> should return <code>409114</code>.');"
+        },
+        {
+          "text":
+            "'<code>leftFactorial(17)</code> should return <code>22324392524314</code>.'",
+          "testString":
+            "assert.equal(leftFactorial(17),22324392524314,'<code>leftFactorial(17)</code> should return <code>22324392524314</code>.');"
+        },
+        {
+          "text":
+            "'<code>leftFactorial(19)</code> should return <code>6780385526348314</code>.'",
+          "testString":
+            "assert.equal(leftFactorial(19),6780385526348314,'<code>leftFactorial(19)</code> should return <code>6780385526348314</code>.');"
+        }
+      ],
+      "id": "5a23c84252665b21eecc7ee0",
+      "challengeType": 5,
+      "releasedOn": "June 30, 2018",
+      "files": {
+        "indexjs": {
+          "key": "indexjs",
+          "ext": "js",
+          "name": "index",
+          "contents": ["function leftFactorial (n) {", "  // Good luck!", "}"],
+          "head": [],
+          "tail": []
+        }
+      }
+    },
+    {
+      "title": "Letter frequency",
+      "description": [
+        "Write a function to count the occurrences of each character in the given string.",
```

Comment:
"Write a function to count the occurrences of each character in a given string.",